ci/pipelines/s3-resource-simple-template.yml

```diff
@@ -0,0 +1,140 @@
+s3-params: &s3-params
+  bucket: ci-pix4d-concourse-third-party
+  region_name: eu-west-1
+  access_key_id: ((concourse_user_access_key))
+  secret_access_key: ((concourse_user_secret_key))
+
+###############################################################################
+
+resource_types:
+
+- name: s3-resource-simple
+  type: docker-image
+  source:
+    repository: docker.internal:5000/((s3-resource-simple-registry))
+    insecure_registries: ['docker.internal:5000']
+    tag: ((branch))-latest
+    username: ((docker_cloud_pix4d_user))
+    password: ((docker_cloud_pix4d_password))
+
+- name: docker-image
+  privileged: true
+  type: docker-image
+  source:
+    repository: concourse/docker-image-resource
+
+###############################################################################
+
+resources:
+
+- name: s3-resource-simple.git
+  type: git
+  source:
+    uri: https://github.com/Pix4D/s3-resource-simple.git
+    ignore_paths: [ci/work-img]
+    branch: ((branch))
+
+- name: s3-resource-simple-work-img.git
+  type: git
+  source:
+    uri: https://github.com/Pix4D/s3-resource-simple.git
+    paths: [ci/work-img]
+    branch: ((branch))
+
```

Comment:
Would you mind adding just a short comment about what the semantic differences are between s3-resource-simple-work-img.docker and s3-resource-final-img.docker? Without any context I am not sure how they are to be used.

Reply:
Even better, name that entirely differently. My understanding is that this is the docker image used to run the test script that just produces a tree of files. It has nothing to do with S3. So my proposal is to name it like `test-fs-tree-producer.docker`.